themes/origins/assets/component-reviews.js

```diff
@@ -0,0 +1,150 @@
+class Reviews extends HTMLElement {
+  static observedAttributes = ["product-id"];
+
+  constructor() {
+    super();
+    this.productId = this.getAttribute("product-id");
+
+    this.total = this.querySelector("[data-total]");
+    this.skeleton = this.querySelector("[data-skeleton]");
+    this.container = this.querySelector("[data-container]");
+    this.percentages = this.querySelectorAll("[data-graph] [data-percentage]");
+
+    this.filter = this.querySelector("[data-filters]");
+    this.filterOptions = this.filter.querySelectorAll("input:not(:first-child)");
+  }
+
+  connectedCallback() {
+    this._render();
+  }
+
+  async _render() {
+    try {
+      const response = await youcanjs.product.fetchReviews(this.productId).data();
+
+      response.length && this.setupItems(response);
+    } catch (error) {
+      console.error(error);
+
+      this.productId && toast.show(error.message, "error");
+    } finally {
+      this.skeleton.remove();
+    }
+  }
+
+  setupItems(items) {
+    const ITEMS_WITH_CONTENT = items.filter((item) => Boolean(item.content));
+    const TOTAL_ITEMS = ITEMS_WITH_CONTENT.length;
```

Comment:
capital constants are usually used with literal value assignments, we can use `totalItems` here instead
```suggestion
    const totalItems = ITEMS_WITH_CONTENT.length;
```